README.md

```diff
@@ -415,6 +434,12 @@ Roughly:
 
 PRs that have been approved by an author listed in the definition's header are usually merged more quickly; PRs for new definitions will take more time as they require more review from maintainers. Each PR is reviewed by a TypeScript or Definitely Typed team member before being merged, so please be patient as human factors may cause delays. Check the [New Pull Request Status Board](https://github.com/DefinitelyTyped/DefinitelyTyped/projects/5) to see progress as maintainers work through the open PRs.
 
+#### I'd like to submit a change to a very popular project, why are they treated differently?
+
+For changes to very popular modules, e.g. Node/Express/Jest which have many millions of downloads each per week on npm, the requirements for contributions are a bit higher. 
+These modules require both a sign-off from a DT maintainer, and enthusiastic support from the module owners. The bar for passing this can be quite high, and often PRs can go stale because it doesn't have a champion.
+Changes to these projects can have massive ecosystem effects, and so we treat changes to the with a lot of care.
```

Comment:
I would swap these two sentences.

Reply:
I think that another useful point to mention here is to try to minimize changes, since I've seen people being frustrated with their PRs not getting attention, where in practice they're ignored because of the much-larger-than-could-have-been factor.  (But that's probably good for all popularity levels too.)